activerecord/lib/active_record/associations.rb

```diff
@@ -1601,6 +1601,12 @@ def has_many(name, scope = nil, **options, &extension)
         #
         #   Note that NestedAttributes::ClassMethods#accepts_nested_attributes_for sets
         #   <tt>:autosave</tt> to <tt>true</tt>.
+        # [:touch]
+        #   If true, the associated object will be touched (the updated_at/on attributes set to current time)
+        #   when this record is either saved or destroyed. If you specify a symbol, that attribute
+        #   will be updated with the current time in addition to the updated_at/on attribute.
+        #   Please note that with touching no validation is performed and only the +after_touch+,
+        #   +after_commit+ and +after_rollback+ callbacks are executed.
```

Comment:
Just a few small tweaks:

```suggestion
        #   If true, the associated object will be touched (the +updated_at+ / +updated_on+ attributes set to current time)
        #   when this record is either saved or destroyed. If you specify a symbol, that attribute
        #   will be updated with the current time in addition to the +updated_at+ / +updated_on+ attribute.
        #   Please note that no validation will be performed when touching, and only the +after_touch+,
        #   +after_commit+, and +after_rollback+ callbacks will be executed.
```

Would you mind also applying these changes to the matching [`belongs_to` documentation](https://github.com/rails/rails/blob/b065d5aff96369e69f1f52f994d92a1fe1f3ca2c/activerecord/lib/active_record/associations.rb#L1750-L1755)? :pray: